src/lsp-server/MAINTAINER.md

```diff
@@ -0,0 +1,42 @@
+
+# Maintainer instructions for `mz-lsp-server`
+
+## Testing the Language Server Protocol (LSP)
+
+All the tests for the LSP reside in a single function inside `./tests/test.rs`. To test locally, run:
+
+```
+cargo test -p mz-lsp-server
+```
+
+## Cutting a new release
+
+1. Update the version in [src/lsp-server/Cargo.toml](/src/lsp-server/Cargo.toml).
+
+2. Update the `lsp-server` release notes in the docs with any changes since the last
+   version.
+
+3. Run `cargo run -p mz-lsp-server` once to update `Cargo.lcok`
+
+4. Open a PR with your change and get it merged:
```

Comment:
Does the naming conventions matter for the branch? Or does it only matter for the tag

Reply:
Only matters for the tag. The branch can have any name. I matched the package name to not confuse the relase with the Materialize release or mz release.

Reply:
It only matters for the tag; the branch can have any name. I matched the package name to avoid confusion between the Materialize release or the mz release.